packages/app/src/app/pages/VSCodeAuth/Prompt/index.tsx

```diff
@@ -101,12 +161,13 @@ export const Prompt: FunctionComponent = () => {
 
       <Buttons>
         <Button
-          as="a"
           autoWidth
-          href={deepLink}
           style={{ fontSize: 16, height: 40, width: '100%', marginTop: '1rem' }}
+          onClick={openInVsCode}
```

Comment:
Converting this link to a button because if we navigate to `vscode-insiders://` from a link we cannot catch the exception that happens when the protocol handler is not registered. We use the `openUrl` from `protocol-handlers` to check for that, so we need an `onClick`.